src/Acuminator/Acuminator.Vsix/Tool Windows/CodeMap/ViewModel/CodeMapWindowViewModel.cs

```diff
@@ -12,9 +12,11 @@
 using Acuminator.Utilities.Common;
 using Acuminator.Vsix.Utilities;
 
-
 using ThreadHelper = Microsoft.VisualStudio.Shell.ThreadHelper;
-using System.Windows;
+using static Microsoft.VisualStudio.Shell.VsTaskLibraryHelper;
+
```

Comment:
Unnecessary empty lines

Reply:
I like to separate more complex using statics from a simple list of usings. Imho it improves readability

Reply:
Here we have unnecessary lines between the static using and the namespace keyword. It doesn't separate regular usings from the static using.

Reply:
done